config/storage.yml.example

```diff
@@ -1,34 +1,34 @@
-test:
```

Comment:
this change doesn't make any sense

Reply:
how I can fix it?
I didn't change these files.
When I set up my local database, I deleted them. Then added it back. But the git decided to comment them out as modified.